native/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@codepair/native",
+  "version": "1.0.1",
+  "main": "dist/main.js",
+  "description": "Codepair Native",
+  "author": "yorkie-team",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/minai621/codepair"
+  },
+  "license": "Apache-2.0",
+  "scripts": {
```

Comment:
I think we can keep the scripts simpler by using `electron-forge`. It’s recommended in the Electron documentation. Is there a reason we’re not using it?

Reply:
@devleejb 
I tried using electron-forge as a build tool.  However, there were issues with dependency installation when using pnpm. It seems electron-forge doesn't have proper support for pnpm yet.
This is being discussed in the following issue
https://github.com/electron/forge/issues/2633